docs/examples/opencensus-shim/requirements.txt

```diff
@@ -1,6 +1,6 @@
 opentelemetry-api
 opentelemetry-sdk
-opentelemetry-exporter-jaeger
+opentelemetry-exporter-otlp
 opentelemetry-opencensus-shim
```

Comment:
Do we have this package? Searching in PyPI, it doesn't seem the package is the same as we have in the repo

Reply:
I mean, opentelemetry-opencensus-shim

Reply:
Should that entry be removed ? Please advise

Reply:
It is indeed the same package but the contents have not  been uploaded due to a previous name squatting incident.

Reply:
Got it. So, you can keep it then @arunk1988